src/Valit/Rules/CollectionValitRule.cs

```diff
@@ -57,11 +57,16 @@ IValitResult IValitRule<TObject>.Validate(TObject @object)
             return result;
         }
 
-        private IValitResult ValidatePropertyRules(IEnumerable<IValitRule<TObject>> propertyRules, TObject @object)
-            => ValitRules<TObject>
-                .Create(propertyRules)
-                .WithStrategy(_strategy)
-                .For(@object)
-                .Validate();
+		private IValitResult ValidatePropertyRules(IEnumerable<IValitRule<TObject>> propertyRules, TObject @object)
+			=> ValitRules<TObject>
+				.Create(propertyRules)
+				.WithStrategy(_strategy)
+				.For(@object)
+				.Validate();
+
+        public IEnumerable<IValitRule<TObject>> GetEnsureRules(TObject @object)
+        {
+            return new List<IValitRule<TObject>>{ this };
```

Comment:
No need to initialize a `List` here, we can do `new []{ this };`.
Maybe also use a Expression-bodied member, since we're using it everywhere else?